.github/workflows/build_and_test.yml

```diff
@@ -872,6 +872,8 @@ jobs:
       run: |
         # We need this link because the jekyll build calls `python`.
         ln -s "$(which python3.9)" "/usr/local/bin/python"
+        # We need this link to make sure `python3` points to `python3.9` which contains the prerequisite packages.
+        ln -s "$(which python3.9)" "/usr/local/bin/python3"
```

Comment:
Currently, there are multiple pythons  in the ci env： 
- Python 3.9.19
- Python 3.10.12
 And python3 points to `Python 3.10.12`.